grails-app/conf/Config.groovy

```diff
@@ -1124,9 +1124,8 @@ openboxes {
             label = "customLinks.label"
             defaultLabel = "Custom Links"
             menuItems = [
-                [label: "requestItemCreation.label", defaultLabel: "Request Item Creation", href: "", target: "_blank"], // Fill in href
-                [label: "trainingVideos.label", defaultLabel: "Training Videos", href: "", target: "_blank"], // Fill in href
-                [label: "tableroNuevo.label", defaultLabel: "Tablero Nuevo", href: "/${appName}/tablero"],
+                    [label: "requestItemCreation.label", defaultLabel: "Request Item Creation", href: "https://goo.gl/forms/x1red4Q7fL0ThH253", target: "_blank"],
```

Comment:
@DominikaHatala I think this should not be here.

Reply:
@awalkowiak why not?

Reply:
I meant the value of href, because we don't want to show such links in a public repo.

Reply:
@awalkowiak Yeah sure, I've just updated PR.